# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1361423306 18000
#      Thu Feb 21 00:08:26 2013 -0500
# Node ID c53f8f706c555cfa9706097607b1fd45cf13a434
# Parent  3a90c3f4046d11c268c7161ce45cf4297a4de87e
Reverse logic of libcurl feature test result

* configure.ac (CURLOPT_DIRLISTONLY): Reverse logic of feature test to
clarify reported result.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -856,26 +856,26 @@ save_LIBS="$LIBS"
 CPPFLAGS="$Z_CPPFLAGS $CPPFLAGS"
 LIBS="$Z_LDFLAGS $Z_LIBS $LIBS"
 OCTAVE_CHECK_LIB(curl, cURL,
   [cURL library not found.  The ftp objects, urlread and urlwrite functions will be disabled.],
   [curl/curl.h], [curl_easy_escape])
 if test -z "$warn_curl"; then
   ## Additional check on cURL library that was found
   AC_CACHE_CHECK([for CURLOPT_DIRLISTONLY in curl/curl.h],
-    [octave_cv_header_define_curlopt_dirlistonly],
+    [octave_cv_curl_has_curlopt_dirlistonly],
     [AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
       #include <curl/curl.h>
       ]], [[
       curl_easy_setopt ((CURL*)NULL, CURLOPT_DIRLISTONLY, 0);
       ]])],
-      [octave_cv_header_define_curlopt_dirlistonly=no],
-      [octave_cv_header_define_curlopt_dirlistonly=yes])
+      [octave_cv_curl_has_curlopt_dirlistonly=yes],
+      [octave_cv_curl_has_curlopt_dirlistonly=no])
     ])
-  if test $octave_cv_header_define_curlopt_dirlistonly = yes; then
+  if test $octave_cv_curl_has_curlopt_dirlistonly = no; then
     AC_DEFINE(CURLOPT_DIRLISTONLY, CURLOPT_FTPLISTONLY,
       [Define to the legacy option name if using an older version of cURL.])
   fi
 fi
 LIBS="$save_LIBS"
 CPPFLAGS="$save_CPPFLAGS"
 
 ### Check for either of Graphics/ImageMagick++ libraries
